File: Data Structures & Algorithms/max-water-container/submission-0.py
class Solution:
    def maxArea(self, heights: List[int]) -> int:
        maximum = 0
        i = 0
        while i < len(heights) - 1:
            j = i + 1
            while j < len(heights):
                volume = min(heights[i], heights[j]) * (j - i)
                maximum = max(maximum, volume)
                j += 1
            i += 1
        return maximum

File: Data Structures & Algorithms/power-of-two/submission-0.swift
class Solution {
    func isPowerOfTwo(_ n: Int) -> Bool {
        if n == 1 {
            return true
        }
        if n <= 0 || n % 2 == 1 {
            return false
        }
        return isPowerOfTwo(n / 2)
    }
}
